### PR TITLE
Process and save TPC-DS performance summary data for iterations

### PR DESCRIPTION
**Pull request by HongW2019**
Passed tests:
```
$ cloudtik submit small.yaml tpcds-power-test.scala --conf spark.driver.scaleFactor=1 --conf spark.driver.fsdir="gs://<bucket>"  --conf spark.driver.iterations=3   --jars '$HOME/runtime/benchmark-tools/spark-sql-perf/target/scala-2.12/spark-sql-perf_2.12-0.5.1-SNAPSHOT.jar'
......

Results written to table: 'sqlPerformance' at gs://<bucket>/shared/data/results/tpcds_parquet/1/timestamp=1663227200150
+--------+------+------+------+-----+----+-------+
|   Query|Round1|Round2|Round3|  Max| Min|Average|
+--------+------+------+------+-----+----+-------+
|q19-v2.4|  9.52|  4.05|  2.43| 9.52|2.43|   5.33|
|q20-v2.4|   4.2|  2.61|  2.29|  4.2|2.29|   3.03|
|   Total| 13.72|  6.66|  4.72|13.72|4.72|   8.36|
+--------+------+------+------+-----+----+-------+

Performance summary is saved to gs://<bucket>/shared/data/results/tpcds_parquet/1/timestamp=1663227200150/summary/finalresult.csv
```

### CODE REVIEW

File: tools/benchmarks/spark/scripts/tpcds-power-test.scala [hunk 1]
```diff
@@ -96,13 +96,47 @@ val experiment = tpcds.runExperiment(
...
+val finalResultPath = s"${experiment.resultPath}/summary"
+finalResult.repartition(1).write.option("header", "true").mode("overwrite").csv(finalResultPath)
+
+import org.apache.hadoop.fs.{FileSystem, Path}
+import java.net.URI
+
+val fs = FileSystem.get(URI.create(finalResultPath), sc.hadoopConfiguration)
+val file = fs.globStatus(new Path(s"$finalResultPath/*.csv"))(0).getPath().getName()
+val srcPath=new Path(s"$finalResultPath/$file")
+val destPath= new Path(s"$finalResultPath/finalresult.csv")
```
Comment (HongW2019) [inline tools/benchmarks/spark/scripts/tpcds-power-test.scala:137]: new Path(s"$finalResultPath/summary.csv")

Comment (HongW2019) [reply tools/benchmarks/spark/scripts/tpcds-power-test.scala]: Done.

**Comment by HongW2019** [comment]
After refined the performance result file name, I re-ran the test and it passed as below. @jerrychenhf  Please take a review, thanks.
```
…
Results written to table: 'sqlPerformance' at gs://<bucket>/shared/data/results/tpcds_parquet/1/timestamp=1663574249020
+--------+------+------+------+-----+----+-------+                              
|   Query|Round1|Round2|Round3|  Max| Min|Average|
+--------+------+------+------+-----+----+-------+
|q19-v2.4|  6.77|  4.41|  2.44| 6.77|2.44|   4.54|
|q20-v2.4|  3.91|  2.84|  2.16| 3.91|2.16|   2.97|
|   Total| 10.68|  7.25|   4.6|10.68| 4.6|   7.51|
+--------+------+------+------+-----+----+-------+

Performance summary is saved to gs://<bucket>/shared/data/results/tpcds_parquet/1/timestamp=1663574249020/summary/summary.csv
```

**Comment by jerrychenhf** [comment]
Looks good. Will commit.

**Pull request merged**